### DISCUSSION TITLE
Preview image

### DISCUSSION THREAD

**Discussion by hu-yating** [hugo-themes/toha#526] [Q&A]
Hi, 
I'm trying to change the preview image of my portfolio. Right now it uses one of the logos from skills. 
On the internet I found that I should "Provide a metadata tag for the og:image in the page's head section" with example "<meta name="image" property="og:image" content="[Image URL here]">
But I don't understand where to put this? 

Help is much appreciated

**Comment by hossainemruz** [comment]
For changing preview image for your site, you have to add opengraph hader at `site.yaml` file in your `data/en` folder. Here is an example: https://github.com/hugo-toha/hugo-toha.github.io/blob/bcb1f688fd69ccf0cd6153995b483b714f01c499/data/en/site.yaml#L24

**Reply by hu-yating** [reply]
Hi @hossainemruz, 

Thanks for answering me. I tried your approach in the past and it didn't work at first. I figured out why this was the case.
Reasons for this was because 1) the image size was not optimal for certain sites (e.g. for me this method would work for some sites only). 2) I didn't clear the cache after changing the image. 

So, applying your approach combined with replacing my image with another and adding /?1 to the url to clear the cache did the job for me. 

Thank you!

**Reply by hossainemruz** [reply]
Thanks for sharing.
